packages/website/src/pages/Home.tsx

```diff
@@ -191,10 +148,9 @@ const Layout: React.FC = () => (
     </Section>
 );
 
-const Input: React.FC = () => (
+const FormPages: React.FC = () => (
```

Comment:
I think all the link here should be updated, they still target `#/input/something`